agent/commands/run.go

```diff
@@ -55,14 +58,33 @@ func Run() {
 		config.ConfigureLogger(cfg)
 		l.Debugf("Loaded configuration: %+v", cfg)
 
+		cleanupTmp(cfg.Paths.TempDir, l)
+
 		run(ctx, cfg, configFilepath)
 
+		cleanupTmp(cfg.Paths.TempDir, l)
```

Comment:
don't we remove files on agent stop? I think we should do it on agent stop as well.

Reply:
its done after `run` completes

Reply:
added clear on agent stop